src/core/containers.py

```diff
@@ -34,3 +35,28 @@ def __init__(self, flow_name, task_name):
 
     def as_dict(self):
         return {"flow": self.flow, "task": self.task}
+
+
+class Populate(Container):
+    __slots__ = ["path", "validators", "value"]
+
+    def __init__(self, validators, path=None, value=None, **kwargs):
+        super().__init__(**kwargs)
+        self.path = path
+        self.validators = validators
+        self.value = value
+        if value and path:
```

Comment:
could you move this stuff into  ```_validate_args``` or something like that and call it before setting the values?

Reply:
done